docs/content/preview/yugabyte-cloud/managed-automation/managed-cli.md

```diff
@@ -0,0 +1,696 @@
+---
+title: YugabyteDB Managed CLI
+headerTitle: YugabyteDB Managed CLI
+linkTitle: YugabyteDB Managed CLI
+description: Use YugabyteDB Managed CLI to access YugabyteDB clusters.
+headcontent: Manage cluster and account resources from the command line
+menu:
+  preview_yugabyte-cloud:
+    identifier: managed-cli
+    parent: managed-automation
+    weight: 20
+type: docs
+rightNav:
+  hideH4: true
+---
+
+## Overview
+
+The YugabyteDB Managed Command Line Interface (YBM CLI) is an open source tool that enables you to interact with YugabyteDB Managed accounts using commands in your command-line shell. With minimal configuration, the CLI enables you to start running commands that implement functionality equivalent to that provided by the browser-based YugabyteDB Managed interface from the command prompt in your shell.
+
+You can install the YugabyteDB Managed CLI using any of the following methods:
+
+Using Docker:
+
+```sh
+docker run -it yugabytedb/yugabyte-client ybm-cli -h <hostname> -p <port>
+```
+
+Using Homebrew:
+
+```sh
+brew tap yugabyte/yugabytedb
+brew install ybm-cli
+```
+
+Using a shell script:
+
+```sh
+curl -sSL https://downloads.yugabyte.com/get_ybm_cli.sh | bash
+```
+
+If you have wget, you can use the following:
+
+```sh
+wget -q -O - https://downloads.yugabyte.com/get_ybm_cli.sh | sh
+```
+
+## Setup
+
+Using ybm-cli requires providing, at minimum, an [API key](../managed-apikeys/) and the host address.
+
+You can pass these values as flags. For example:
+
+```sh
+ybm --apikey AWERDFSSS --host cloud.yugabyte.com cluster get
+```
+
+For convenience, configure `ybm-cli` with default values for these flags as follows:
+
+- Using a configuration file called `.ybm-cli.yaml` under your `$HOME` directory. You can use the command `ybm configure` to set up the file. For example:
```

Comment:
Currently, `.ybm-cli.yaml` is the default file name, but user can specify a different file with the flag `--config` for example.
```
ybm --config ~/.ybm-cli-portal.yaml cluster get
```